DiscountDiscover/Models/Deal.m

```diff
@@ -69,6 +71,13 @@ - (NSString *)formatStoreAddressString:(NSDictionary *)dictionary {
     }
 }
 
+- (GMSMarker *)createMarker {
+    GMSMarker *marker = [GMSMarker markerWithPosition:self.storeCoordinate];
```

Comment:
As a stretch goal, it could be nice to easily differentiate deals on the map by using different marker colors, for example depending on the discount % or on the deal category.